examples/c/nonlinear_chain_ocp_nlp.c

```diff
@@ -1231,8 +1237,12 @@ int main()
 	//
 	external_function_casadi_create_array(NN, impl_ode_jac_x_xdot_u);
 
-	// discrete model
-	external_function_casadi_create_array(NN, erk4_casadi);
+	if (NMF<4)
```

Comment:
if you throw an error before, you don't need this if here

Reply:
i changed it to be as you wrote